app/src/main/java/org/projectbuendia/client/ui/chart/ChartRenderer.java

```diff
@@ -36,6 +33,7 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
No, because we could use `lastChart.equals(chart.name)` below as before, but I like `Objects.equals` for non-final variables because it's `null`-safe as well :)